pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -149,15 +149,40 @@ func (p *ProjectService) tlsSecretName() string {
 
 // getKubernetesUpdateStrategy gets update strategy for compose project service
 // Note: it only supports `parallelism` and `order`
-// @todo add label support for update strategy!
 func (p *ProjectService) getKubernetesUpdateStrategy() *v1apps.RollingUpdateDeployment {
+	// The update strategy should only rely on settings defined in a k8s extension. As the settings have
+	// already been inferred from the compose.go Deploy.UpdateConfig block.
+	//
+	// *****For now, as we can only configure settings for RollingUpdateMaxSurge, this is the only
+	// strategy that is enabled be default.*****
+	//
+	// An extra setting, e.g. strategy, should be defined to qualify which UpdateStrategy to use. As before,
+	// 'strategy' can be inferred from the Deploy block with defaults added as required.
+	//
+	// The 'strategy' setting will definitely be required when MaxUnavailable config is made available to a k8s extension.
+	// E.g. here's a yaml sample of the proposal:
+	// rollingUpdate:
+	//   strategy: surge
```

Comment:
If we ever introduce the strategy toggle then we should follow k8s strategies list with `RollingUpdate` being a default. We might need to rethink the way the `strategy` part gets expressed in the k8s extension. It could be as in example below (or similar):
```yaml
strategy:  # will cover Pod replacement / Statefulset update strategies perhaps
  type: RollingUpdate
  rollingUpdate: # there is no additional settings for `Replace` strategy currently.
    maxUnavailable: 2
    maxSurge: 2
```
However, the above is applicable to [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) and we also should rethink this in context of [StatefulSet update strategies](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies).

Reply:
I'll get rid of the comments.